### ISSUE TITLE
TIMELINE: ether-painters.js -> timeZone bug

### ISSUE THREAD

**Issue by GoogleCodeExporter** [berusch/simile-widgets-old#52]
```
In file ether-painter.js all calls to 
SimileAjax.DateTime.incrementByInterval do NOT
include the timeZone parameter, causing incorrect painted ether lines 
and labels. But timeZone should be included in the function call.

This can be easily fixed by adding the band's timeZone in the function 
call, e.g:

SimileAjax.DateTime.incrementByInterval(date, p._unit, p._band.getTimeZone
());

Regards
```

Original issue reported on code.google.com by `HylkeSpr...@gmail.com` on 5 Jan 2009 at 8:11

**Comment by GoogleCodeExporter** [comment]
```
This may be by design. I will ask David H for his opinion (he's the original 
author). 
Thank you for the report.

```

Original comment by `larryklu...@gmail.com` on 7 Jan 2009 at 2:48

**Comment by GoogleCodeExporter** [comment]
Original comment by `stefano.mazzocchi@gmail.com` on 25 Mar 2009 at 7:01
- Added labels: **Project-Timeline**

**Comment by GoogleCodeExporter** [comment]
Original comment by `stefano.mazzocchi@gmail.com` on 25 Mar 2009 at 7:15
- Changed title: **TIMELINE: ether-painters.js -> timeZone bug**
